Sources/StreamChat/APIClient/Endpoints/Payloads/PollsPayloads.swift

```diff
@@ -0,0 +1,251 @@
+    let id: String
+    let name: String
+    let updatedAt: Date
+    let voteCount: Int
+    let votingVisibility: String
+    let options: [PollOptionResponseData?]
+    let ownVotes: [PollVoteResponseData?]
+    var custom: [String: RawJSON]?
+    let latestVotesByOption: [String: [PollVoteResponseData?]]
+    var voteCountsByOption: [String: Int]?
+    var isClosed: Bool?
+    var maxVotesAllowed: Int?
+    var createdBy: UserPayload?
+
+    init(
+        allowAnswers: Bool,
+        allowUserSuggestedOptions: Bool,
+        answersCount: Int,
+        createdAt: Date,
+        createdById: String,
+        description: String,
+        enforceUniqueVote: Bool,
+        id: String,
+        name: String,
+        updatedAt: Date,
+        voteCount: Int,
+        votingVisibility: String,
+        options: [PollOptionResponseData?],
+        ownVotes: [PollVoteResponseData?],
+        custom: [String: RawJSON]?,
+        latestVotesByOption: [String: [PollVoteResponseData?]],
+        voteCountsByOption: [String: Int],
+        isClosed: Bool? = nil,
+        maxVotesAllowed: Int? = nil,
+        createdBy: UserPayload? = nil
+    ) {
+        self.allowAnswers = allowAnswers
+        self.allowUserSuggestedOptions = allowUserSuggestedOptions
+        self.answersCount = answersCount
+        self.createdAt = createdAt
+        self.createdById = createdById
+        self.description = description
+        self.enforceUniqueVote = enforceUniqueVote
+        self.id = id
+        self.name = name
+        self.updatedAt = updatedAt
+        self.voteCount = voteCount
+        self.votingVisibility = votingVisibility
+        self.options = options
+        self.ownVotes = ownVotes
+        self.custom = custom
+        self.latestVotesByOption = latestVotesByOption
+        self.voteCountsByOption = voteCountsByOption
+        self.isClosed = isClosed
+        self.maxVotesAllowed = maxVotesAllowed
+        self.createdBy = createdBy
+    }
+    
+    enum CodingKeys: String, CodingKey, CaseIterable {
+        case allowAnswers = "allow_answers"
+        case allowUserSuggestedOptions = "allow_user_suggested_options"
+        case answersCount = "answers_count"
+        case createdAt = "created_at"
+        case createdById = "created_by_id"
+        case description
+        case enforceUniqueVote = "enforce_unique_vote"
+        case id
+        case name
+        case updatedAt = "updated_at"
+        case voteCount = "vote_count"
+        case votingVisibility = "voting_visibility"
+        case options
+        case ownVotes = "own_votes"
+        case custom
+        case latestVotesByOption = "latest_votes_by_option"
+        case voteCountsByOption = "vote_counts_by_option"
+        case isClosed = "is_closed"
+        case maxVotesAllowed = "max_votes_allowed"
+        case createdBy = "created_by"
+    }
+}
+
+struct PollOptionResponse: Decodable {
+    let duration: String
+    let pollOption: PollOptionResponseData
```

Comment:
From the comments above, this would be instead:
```suggestion
    let pollOption: PollOptionPayload
```